docs/conf.py

```diff
@@ -281,6 +281,7 @@
     "numpy": ("https://numpy.org/doc/stable", None),
     "scipy": ("https://docs.scipy.org/doc/scipy", None),
     "matplotlib": ("https://matplotlib.org/stable/", None),
+    "jax": ("https://jax.readthedocs.io/en/latest", None),
```

Comment:
```suggestion
    "jax": ("https://jax.readthedocs.io/en/stable", None),
```

Reply:
@arjxn-py I agree with this one, so I am just going to merge it to re-trigger